--- libmusicbrainz-2.1.2/python/musicbrainz.py.wiget	2006-04-11 18:21:19.000000000 +0200
+++ libmusicbrainz-2.1.2/python/musicbrainz.py	2006-04-11 18:23:34.000000000 +0200
@@ -37,22 +37,18 @@
         else:
            raise MusicBrainzError, "Unknown platform: " + sys.platform 
   
-    path = "/usr/lib/" + lib
-    if os.access(path, os.F_OK):
-       return path;
-
-    path = "/usr/local/lib/" + lib
-    if os.access(path, os.F_OK):
-       return path;
+    return lib
 
-    raise MusicBrainzError, "Cannot find MusicBrainz share library: " + lib
 
-if sys.platform == "win32":
-    mbdll = cdll.LoadLibrary("libmusicbrainz.dll")
-    mbdll.mb_WSAInit.argtypes = [c_int]
-    mbdll.mb_WSAStop.argtypes = [c_int]
-else:
-    mbdll = cdll.LoadLibrary(findLibrary())
+try:
+    if sys.platform == "win32":
+        mbdll = cdll.LoadLibrary("libmusicbrainz.dll")
+        mbdll.mb_WSAInit.argtypes = [c_int]
+        mbdll.mb_WSAStop.argtypes = [c_int]
+    else:
+        mbdll = cdll.LoadLibrary(findLibrary())
+except OSError:
+    raise MusicBrainzError, "Cannot find MusicBrainz share library: " + lib
 
 class mb:
     mbdll.mb_New.argtypes = []
