diff -Nur libmusicbrainz-2.0.1.orig/lib/parse.h libmusicbrainz-2.0.1/lib/parse.h
--- libmusicbrainz-2.0.1.orig/lib/parse.h	2000-10-26 14:19:46.000000000 +0200
+++ libmusicbrainz-2.0.1/lib/parse.h	2003-05-16 23:21:06.000000000 +0200
@@ -38,7 +38,7 @@
 
 #include "errors.h"
 #include "types.h"
-#include "../expat/xmlparse/xmlparse.h"
+#include <expat.h>
 
 typedef map<string, string > AttrMap;
 
--- libmusicbrainz-2.0.2/lib/rdfparse.h.orig	2003-02-28 06:12:12.000000000 +0100
+++ libmusicbrainz-2.0.2/lib/rdfparse.h	2003-07-19 23:29:13.000000000 +0200
@@ -73,11 +73,7 @@
 extern "C" {
 #endif
 
-#ifdef RDFPARSE_INCLUDE_XMLPARSE
-#include "xmlparse.h"
-#else
-#include "../expat/xmlparse/xmlparse.h"
-#endif
+#include <expat.h>
 
 #ifndef RDFPARSEAPI
 #define RDFPARSEAPI /* as nothing */
